1249/en.md

```diff
@@ -0,0 +1,95 @@
+# LOJ 1249 - Chocolate Thief
+
+### Problem statement
+In this problem, each test case contains `n` number of lines. Each line contains:
+ `Name(n) length(l),breadth(b) height(h)`
+
+Denoting that the person with name(n) ate the cake of the length(l),breadth(b) & height. Each information separated by a `whitespace` .Each person should eat the same amount of cake `in volume` i.e each `l*b*h` should be the same.
+However, one single `thief` stole a portion from another `victim`, given that in all cases, 
+```
+number of thief = number of victim = 1
+```
+We need to print out the `name` of the `thief` & `victim` in a particular testcase.
+
+**Example Output:** `Case 1: mun took chocolate from all` 
+
+### Approach
+
+Basically we need to find out the name associated with `higher than average volume`  and `lower than average volume` 
+of consumed cake. I used this data-structure to solve the problem:
+```js
+const tab = {
+    [key]: number // this denotes the volume of the consumed cake
+    {
+        name: string  // stored the name, not a necessary field in case of non-thiefs/non-victims
+        freq: number // stored the number of people with the same volume i.e, the key
+    }
+}
+```
+
+- If there is only single key, it means that everybody got the same volume of that cake
+- Otherwise, we find out the **minimum** key and the **maximum** key, along with the associated **name**. Then print out the name of the **victim** and the **thief**
+
+### Javascript
+-----
+```js
+const readline = require('readline')
+const rl = readline.createInterface({
+    input:process.stdin
+})
+const userInput = []
+rl.on('line',(values)=>{
+    userInput.push(values)
+})
+
+
+rl.on('close',()=>{
+    const noOfCases = parseInt(userInput[0])
+    let inputPtr = 1
+    for(let i=0;i<noOfCases;i++){
+        let victim = ''
+        let thief = ''
+        let cases = parseInt(userInput[inputPtr])
+        let tab = {}
+        for(let i = inputPtr+1 ; i< cases+inputPtr+1;i++){
+            let temp = userInput[i].split(" ")
```

Comment:
it would be better if you could split your code a bit. 
```
const l = parseInt(temp[temp.length-1]);
const h = ...
```

in this way code will be more readable.